osu.Game/Overlays/Comments/CommentEditor.cs

```diff
@@ -167,84 +168,75 @@ private void load(OverlayColourProvider colourProvider)
+                Add(spinner = new LoadingSpinner
+                {
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Size = new Vector2(14),
+                    Depth = -2,
+                });
             }
 
             protected override void LoadComplete()
             {
                 base.LoadComplete();
-                IsBlocked.BindValueChanged(onBlockedStateChanged, true);
+                IsBlocked.BindValueChanged(e =>
+                {
+                    Enabled.Value = !IsLoading && !e.NewValue;
+                }, true);
             }
 
-            private void onBlockedStateChanged(ValueChangedEvent<bool> isBlocked)
+            protected override SpriteText CreateText() => text = new OsuSpriteText
             {
-                drawableText.FadeColour(isBlocked.NewValue ? colourProvider.Foreground1 : Color4.White, duration, Easing.OutQuint);
-                background.FadeTo(isBlocked.NewValue ? 0 : 1, duration, Easing.OutQuint);
-            }
+                AlwaysPresent = true,
+                Anchor = Anchor.Centre,
+                Origin = Anchor.Centre,
+                Font = OsuFont.GetFont(size: 12, weight: FontWeight.Bold),
+                Margin = new MarginPadding { Horizontal = 20 },
+            };
 
-            protected override Drawable CreateContent() => new CircularContainer
+            protected override bool OnClick(ClickEvent e)
             {
-                Masking = true,
-                Height = 25,
-                AutoSizeAxes = Axes.X,
-                Children = new Drawable[]
+                if (!Enabled.Value)
+                    return false;
+
+                try
                 {
-                    blockedBackground = new Box
-                    {
-                        RelativeSizeAxes = Axes.Both
-                    },
-                    background = new Box
-                    {
-                        RelativeSizeAxes = Axes.Both,
-                        Alpha = 0
-                    },
-                    drawableText = new OsuSpriteText
-                    {
-                        AlwaysPresent = true,
-                        Anchor = Anchor.Centre,
-                        Origin = Anchor.Centre,
-                        Font = OsuFont.GetFont(size: 12, weight: FontWeight.Bold),
-                        Margin = new MarginPadding { Horizontal = 20 },
-                        Text = text,
-                    }
+                    return base.OnClick(e);
                 }
-            };
-
-            protected override void OnLoadStarted() => drawableText.FadeOut(duration, Easing.OutQuint);
-
-            protected override void OnLoadFinished() => drawableText.FadeIn(duration, Easing.OutQuint);
+                finally
+                {
+                    // run afterwards as this will disable this button.
+                    IsLoading = true;
```

Comment:
This is just weird.

Reply:
Left from existing implementation. There is no usages of this so i agree that it shouldn't exist.